text/game/abilities.stringtable

```diff
@@ -4471,7 +4471,7 @@
     </Entry>
     <Entry>
       <ID>911</ID>
-      <DefaultText>Weist den tierischen Gefährten des Waldläufers an, zu einem Ziel zu preschen und es Niedergeschlagen wird.</DefaultText>
+      <DefaultText>Weist den tierischen Gefährten des Waldläufers an, zu einem Ziel zu preschen und es niederzuwerfen.</DefaultText>
```

Comment:
Die Schlüsselbegriffe würde ich gerne drinne lassen, weil sie einen Link zur Enzyklopädie darstellen. In dem Fall geht "spielregel-konforme Beschreibung" vor gutem Ausdruck, finde ich.